lib/global-styles.php

```diff
@@ -148,7 +148,7 @@ function gutenberg_experimental_global_styles_get_user_cpt_id() {
  */
 function gutenberg_experimental_global_styles_get_core() {
 	return gutenberg_experimental_global_styles_get_from_file(
-		dirname( dirname( __FILE__ ) ) . '/experimental-default-global-styles.json'
```

Comment:
This was moved from the top-level to `lib/global-styles` as per [this comment](https://github.com/WordPress/gutenberg/pull/19883#discussion_r392226550).